model/src/form/form-audit/types.ts

```diff
@@ -130,10 +129,19 @@ export interface FormUploadedMessageData extends FormMessageDataBase {
   changes: ChangesMessageData<FormUploadedChanges>
 }
 
-export type FormChangeSet = IChange
+export interface FormDefinitionS3Meta {
+  fileId?: string
```

Comment:
I think these 3 props should not be `?`. According to the schema they're `string().required()`